platform/windows/display_server_windows.cpp

```diff
@@ -2995,13 +2995,17 @@ DisplayServerWindows::~DisplayServerWindows() {
 		SetWindowLongPtr(windows[MAIN_WINDOW_ID].hWnd, GWLP_WNDPROC, (LONG_PTR)user_proc);
 	};
 
-	if (windows.has(MAIN_WINDOW_ID)) {
+	// Destroy all remaining windows in windows, even though they should already been destroyed,
+	// but sometimes that doesn't happen so let's delete them all here, since we are crashing
+	// the Application after that constructer because there are still callbacks registered and called
+	// by the OS
+	for (Map<int, WindowData>::Element *window = windows.front(); window; window = window->next()) {
+		WARN_PRINT("Open Window detected: " + window->key());
 #ifdef VULKAN_ENABLED
 		if (rendering_driver == "vulkan") {
 			context_vulkan->window_destroy(MAIN_WINDOW_ID);
```

Comment:
This is going to run multiple times if there's more than one window, that doesn't sound like a good idea.